src/navigators/createMaterialTopTabNavigator.js

```diff
@@ -19,12 +19,42 @@ type Props = InjectedProps & {
   tabBarOptions?: TabBarOptions,
 };
 
-class TabView extends React.PureComponent<Props> {
+type State = {
+  index: number,
+  isSwiping: boolean,
+  loaded: Array<number>,
+  transitioningFromIndex: ?number,
+};
+
+class TabView extends React.PureComponent<Props, State> {
   static defaultProps = {
     // fix for https://github.com/react-native-community/react-native-tab-view/issues/312
     initialLayout: Platform.select({
       android: { width: 1, height: 0 },
     }),
+    animationEnabled: true,
+  };
+
+  static getDerivedStateFromProps(nextProps, prevState) {
```

Comment:
Also, need to use `react-lifecycles-compat` here to support older React versions.

Reply:
Oops, done.